remo/settings.py

```diff
@@ -64,7 +64,7 @@
     'csp.middleware.CSPMiddleware',
     'remo.base.middleware.RegisterMiddleware',
     'waffle.middleware.WaffleMiddleware',
-    'mozilla_django_oidc.middleware.RefreshIDToken',
+    # 'mozilla_django_oidc.middleware.RefreshIDToken',
```

Comment:
DO you forgotten to uncomment this before to push it?

Reply:
Yep, totally did not want to do that. Will revert this ASAP. Does the rest look good to you?